packages/jbrowse-web/src/Loader.test.tsx

```diff
@@ -66,7 +66,7 @@ describe('<Loader />', () => {
       </ErrorBoundary>,
     )
     expect(
-      await findByText('Error: HTTP 404 fetching doesNotExist.json', {
+      await findByText('Could not find config "doesNotExist.json"', {
```

Comment:
I think it should report the actual error with the status code

Reply:
Fixed